packages/wix-storybook-utils/src/Sections/AnchoredTitle/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useRef } from 'react';
+import addons from '@storybook/addons';
+import copy from 'copy-to-clipboard';
+import ReactTooltip from 'react-tooltip';
+import { LinkSmall, ConfirmSmall } from 'wix-ui-icons-common';
+
+import Markdown from '../../Markdown';
+import styles from './styles.scss';
+
+const CopyToClipboard = () => <div className={styles.tooltipContent}>{translations.copyLink}</div>
+const CopiedToClipboard = () => <div className={styles.tooltipContent}><ConfirmSmall/>{translations.linkCopiedToClipboard}</div>
+
+const translations = {
+  copyLink: 'Copy Link',
+  linkCopiedToClipboard: 'Copied',
+};
+
+export const AnchoredTitle = ({ title }) => {
+  const [tooltipComponent, setTooltipComponent] = useState(CopyToClipboard)
```

Comment:
Its a bit unusual to switch components in state. Lets better do the usual way where you render the div always and then only switch the other parts.

```suggestion
  const [copied, setCopied] = useState(false)
```